src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -112,6 +112,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('registry_name', completer=None)
         c.argument('deployment_name', validator=None)
         c.argument('location', validator=get_default_location_from_resource_group)
```

Comment:
One dumb question: Is this PR adding log analytics workspace creation upon registry creation?

Reply:
@msyihchen 
It looks like you have to create a Log analytics workspace first. Then hook up a container registry with the workspace while creating.

`az acr create -n <containerRegName> --workspace <workspaceName> --sku basic`

```
--workspace [Preview] : Name or ID of the Log Analytics workspace to send registry
                            diagnostic logs to. All events will be enabled. You can use "az monitor
                            log-analytics workspace create" to create one. Extra billing may apply.
```